### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by anastasijakundas**
https://task2-nodejs.herokuapp.com/api/films/categories

### CODE REVIEW

File: routes/films.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+const express = require("express");
+const bodyParser = require("body-parser");
+const filmCategories = require('./filmCategories');
+const router = express.Router();
+var urlencodedParser = bodyParser.urlencoded({ extended: false })
```
Comment (rkhramiankou) [inline routes/films.js]: const

File: routes/films.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+router
+.get("/", function(req, res) {
+    res.send([])
+})
+.post("/", urlencodedParser, function(req, res) {
```
Comment (rkhramiankou) [inline routes/films.js]: urlencodedParser should be registered with router.use instead of straight param use

**Comment by anastasijakundas** [comment]
https://task2-nodejs.herokuapp.com/api/films/categories

File: routes/films.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+const express = require("express");
+const bodyParser = require("body-parser");
+const filmCategories = require('./filmCategories');
+const router = express.Router();
+const urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+router
+.use(bodyParser.urlencoded({ extended: true }))
```
Comment (rkhramiankou) [inline routes/films.js]: move to app level

File: routes/filmCategories.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+
+const express = require("express");
+const bodyParser = require("body-parser");
+const router = express.Router();
+var urlencodedParser = bodyParser.urlencoded({ extended: false })
+
+router
+.use(bodyParser.urlencoded({ extended: true }))
```
Comment (rkhramiankou) [inline routes/filmCategories.js]: to app level

File: routes/films.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+
+const express = require("express");
+const bodyParser = require("body-parser");
+const filmCategories = require('./filmCategories');
+const router = express.Router();
+const urlencodedParser = bodyParser.urlencoded({ extended: false })
```
Comment (rkhramiankou) [inline routes/films.js]: unused variable?

**Pull request merged**